### PR TITLE
Description UI change to Callout

### PR DESCRIPTION
**Pull request by vandyliu**
[AB#34026](https://microsoftgarage.visualstudio.com/002806e2-ebaa-4672-9d2e-5fe5d29154ef/_workitems/edit/34026)

# Description:
- Note: my vscode color settings are a little messed up on my computer so the GIF colors may look off so take a look for yourselves
- Changed description to a callout
- Removed scroll from sensor modal (so that there is only one scroll bar: the main one)
Light Mode Change
High Contrast Mode Change
Dark  Mode Change

## Type of change
- [ ] New feature (non-breaking change which adds functionality)

# Limitations:

Please describe limitations of this PR

# Testing:

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration
- Manual testing
- [ ] Test A
- [ ] Test B

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] My code has been formatted with npm run format and passes the checks in npm run check
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by xnkevinnguyen** [comment]
Awesome work Vandy! Good work on the frontend side :)

**Review by andreamah — APPROVED** [review]
Looks amazing! Thanks for all of your hard work 😊

**Comment by nasadigital** [comment]
This is how it looks for me:
i_before
After hovering:

Any idea why this is happening?

File: src/view/styles/ToolBar.css [hunk 1]
```diff
@@ -56,24 +56,79 @@
     width: 320px;
 }
 
+.info-icon {
+    background: var(--vscode-debugToolBar-background);
+    border-color: var(--vscode-highContrastButtonBorderOverride-color);
+    color: var(--vscode-badgeForegroundOverride);
+    border-width: 1px;
+    border-style: solid;
+    padding: 0px 4px 0px 4px;
```
Comment (nasadigital) [inline src/view/styles/ToolBar.css]: _&ast;nit&ast;_
You can do `padding: 0px 4px;` which is equivalent to `padding: 0px 4px 0px 4px;`

File: src/view/components/toolbar/ToolBar.tsx [hunk 2]
```diff
@@ -150,22 +165,40 @@ class ToolBar extends React.Component<IProps, IToolbarState, any> {
...
+                            <FormattedMessage id={content.descriptionText} />
+                        </div>
+                    </Callout>
+                )}
                 <div className="try_area">
                     <br />
-                    <span className="description">
+                    <span className="try-it-description">
                         <FormattedMessage id={content.tryItDescription} />
                     </span>
```
Comment (nasadigital) [inline src/view/components/toolbar/ToolBar.tsx]: This text is very difficult to read, it blends in the background 😭

File: src/view/components/toolbar/ToolBar.tsx [hunk 3]
```diff
@@ -150,22 +165,40 @@ class ToolBar extends React.Component<IProps, IToolbarState, any> {
...
+                    >
+                        <div className="description">
+                            <FormattedMessage id={content.descriptionText} />
+                        </div>
+                    </Callout>
+                )}
                 <div className="try_area">
                     <br />
-                    <span className="description">
+                    <span className="try-it-description">
```
Comment (nasadigital) [inline src/view/components/toolbar/ToolBar.tsx:203]: This text is very difficult to read, it blends in the background 😭

**Comment by vandyliu** [comment]
> This is how it looks for me:
> i_before
> After hovering:
> 
> Any idea why this is happening?

@nasadigital
Did you run `npm i` before compiling? That's the only thing I can think of right now.

**Comment by nasadigital** [comment]
@vandyliu Yup, tried it again but it looks the same.
I ran:
```
$ npm install
$ npm run compile
```
followed by "Run Extension" in VS Code.
Do you know if this issue is likely because of my setup or if it will be something more widespread?
By the way, the tooltip shows the description properly it's only the icon that is not showing.

**Comment by xnkevinnguyen** [comment]
> @vandyliu Yup, tried it again but it looks the same.
> I ran:
> 
> ```
> $ npm install
> $ npm run compile
> ```
> 
> followed by "Run Extension" in VS Code.
> Do you know if this issue is likely because of my setup or if it will be something more widespread?
> By the way, the tooltip shows the description properly it's only the icon that is not showing.

Sometimes Vscode seems to not load some elements on the webview correctly. Try closing vscode and reopening it if that's not already done.

**Comment by nasadigital** [comment]
> Sometimes Vscode seems to not load some elements on the webview correctly. Try closing vscode and reopening it if that's not already done.

@xnkevinnguyen That's interesting. I reopened VS Code multiple times, but should I try uninstall -> install?

File: src/view/styles/ToolBar.css (hunk 1 above)
Comment (vandyliu) [reply src/view/styles/ToolBar.css]: updated

File: src/view/components/toolbar/ToolBar.tsx (hunk 3 above)
Comment (nasadigital) [reply src/view/components/toolbar/ToolBar.tsx:203]: My bad, I meant this text:
Annotation 2020-03-18 125448

**Review by nasadigital — APPROVED** [review]
LGTM, Great job!

File: src/view/components/toolbar/ToolBar.tsx (hunk 3 above)
Comment (vandyliu) [reply src/view/components/toolbar/ToolBar.tsx:203]: I noticed that too. I'll take a look into it.

Comment (vandyliu) [reply src/view/components/toolbar/ToolBar.tsx:203]: Updated, let me know what you think now.

Comment (nasadigital) [reply src/view/components/toolbar/ToolBar.tsx:203]: Much better, thank you!

**Review by nasadigital — APPROVED** [review]

**Pull request merged**